lib/puppet-lint.rb

```diff
@@ -225,7 +227,7 @@ def run
 
   # Public: Print any problems that were found out to stdout.
   #
-  # Returns nothing.
+  # Returns an array of problems. (Can be empty depending on configuration!)
```

Comment:
Is this still true? Or should this now simply be an array.

Reply:
Good spot.  Fixed.